doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,41 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Load Balancer platform service.
+
++------------+-----------------------------------------------------------------------+-------------------------------------------------------------------------+
```

Comment:
- Grid between the rows is missing
- Cell separator missing from the first column in the 2nd line of pas.lb.007
- Cell separator is missing from the first column in pas.lb.013 - pas.lb.017
- Can you please wrap it to 120 chars?